sampleExtension/tooltipPage/tooltip.css

```diff
@@ -10,4 +10,13 @@ body {
 .btn-group-vertical > button {
     margin-bottom:10px;
     border-radius: 7px !important;
-}
\ No newline at end of file
+}
+
+#loginBtn {
+    visibility: hidden;
+}
+
+#logoutBtn {
+    visbility: hidden;
+}
+
```

Comment:
this was removed because it has no functionality anymore, dunno why you added this.